From 201b36614907584a0d446975bddbb11a495540ed Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Sat, 22 Feb 2014 17:47:44 +0100
Subject: [PATCH] examples.pro: include server-buffer only when building with
 opengles2

* it's using glBindBuffer in
  server-buffer/client/serverbufferrenderer.cpp

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
Change-Id: I8412dfd4ebb95c147328ac6e4dfff14a0cff4e78
---
 examples/examples.pro | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/examples/examples.pro b/examples/examples.pro
index 2d4cd1b..fb2497e 100644
--- a/examples/examples.pro
+++ b/examples/examples.pro
@@ -13,5 +13,7 @@ contains(CONFIG, wayland-compositor) {
         SUBDIRS += qml-compositor
     }
 
-    SUBDIRS += server-buffer
+    contains(QT_CONFIG, opengles2) {
+        SUBDIRS += server-buffer
+    }
 }
-- 
1.8.5.3

